README.md

```diff
@@ -341,6 +341,22 @@ been prepared for reliable benchmarking.
 Note that you should not collect results intended for publication with
 `--develop`.
 
+## Re-running Part of your Experiment
+
+You can use the `--strip-results` mode to strip results from your result file.
+Once stripped, you can run with `--resume` to re-collect the results you
+stripped.
```

Comment:
Maybe add a note to say why / when it is reasonable to do something like this. Generally, running all the benchmarks in one run would be preferable...